tests/boardfarm_plugins/boardfarm_sample_plugin/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+SPDX-License-Identifier: BSD-2-Clause-Patent
+SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+This code is subject to the terms of the BSD+Patent license.
+See LICENSE file for more details.
+"""
```

Comment:
(nitpick) if you do it like this, it will be the module (or in this case, package) docstring. We generally don't want to use the license header for the docstring; instead, we'd want an actual docstring. Therefore:
```suggestion
# SPDX-License-Identifier: BSD-2-Clause-Patent
# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
# This code is subject to the terms of the BSD+Patent license.
# See LICENSE file for more details.
```